server/lib/notifications.js

```diff
@@ -208,6 +209,31 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+export async function notifyAdminsAndAccountantsOfCollective(CollectiveId, activity, options = {}) {
+  debug('notify admins and accountants of CollectiveId', CollectiveId);
+  const collective = await models.Collective.findByPk(CollectiveId);
+  if (!collective) {
+    throw new Error(
+      `notifyAdminsAndAccountantsOfCollective> can't notify ${activity.type}: no collective found with id ${CollectiveId}`,
+    );
+  }
+
+  let usersToNotify = [];
+  if (collective.type === CollectiveType.USER && !collective.isIncognito) {
+    // Incognito profiles rely on the `Members` entry to know which user it belongs to
+    usersToNotify = [await collective.getUser()];
+  } else {
+    usersToNotify = await collective.getMembersUsers({ role: [roles.ACCOUNTANT, roles.ADMIN] });
```

Comment:
What about Events and Projects? We implicitly rely on the the one from the parent.

And to anticipate, maybe in the future Events and Projects will be allowed to have extra ADMINs too.

Reply:
We
Was